maven_plugin/src/main/java/io/github/chains_project/maven_lockfile/data/LockFile.java

```diff
@@ -32,6 +32,7 @@ public class LockFile {
     private final VersionNumber version;
 
     @SerializedName("lockFileVersion")
+    @SuppressWarnings("FieldMayBeFinal")
     private int lockfileVersion = 1; // TODO: we normally should create an enum with Name -> Numbers
```

Comment:
```suggestion
    private final int lockfileVersion = 1; // TODO: we normally should create an enum with Name -> Numbers
```

Reply:
Created a new PR from bug-fix/workflow-fix, and merged it as all the checks passed.

Reply:
Sounds good!